src/Tribe/Models/Post_Types/Event.php

```diff
@@ -173,6 +173,14 @@ protected function build_properties( $filter ) {
 			$end_site           = $end_date_object->setTimezone( $site_timezone );
 			$use_event_timezone = Timezones::is_mode( Timezones::EVENT_TIMEZONE );
 
+			// Determine if current datetime is between event start and end dates.
+			$is_now             = Dates::range_coincides(
+				$now->format( 'U' ),
+				$now->add( $one_second )->format( 'U' ),
+				$start_date_utc_object->format( 'U' ),
+				$end_date_utc_object->format( 'U' )
+			);
```

Comment:
make this into a callback method no? To avoid caching problems.

Reply:
Smart